lib/custom_wizard/realtime_validation.rb

```diff
@@ -0,0 +1,35 @@
+class CustomWizard::RealtimeValidation
+    cattr_accessor :types
+      @@types ||= {
+        similar_topics: { types: [:text], component: "suggested-validator", backend: true, required_params: [] }
+      }
+
+      class SimilarTopic
```

Comment:
@fzngagan This should be in a seperate file, e.g. ``lib/custom_wizard/realtime_validation/similar_topic.rb``. Again, imagine we're going to have multiple validations. They won't all be in this file. We need to start how we intend to proceed.